### PR TITLE
[TAS-1467] ✨ Implement new home page

### PR DESCRIPTION
**Pull request by nwingt**
localhost_3000_zh-Hant

### CODE REVIEW

File: src/pages/index.vue [hunk 1]
```diff
@@ -1,1643 +1,640 @@
...
+                {{
+                  $t('about_nft_book_section_features_ownership_description')
+                }}
+              </p>
+            </a>
+          </li>
+          <li>
+            <a
+              class="flex flex-col items-center"
+              href="https://blog.like.co/zh/%E5%85%A7%E5%AE%B9%E6%84%9B%E5%A5%BD%E8%80%85%E7%9A%84%E5%BB%A3%E5%A0%B4-liker-land-%E6%96%B0%E5%8A%9F%E8%83%BD/"
```
Comment (williamchong) [inline src/pages/index.vue:404]: Should this be locale sensitive? en version is https://blog.like.co/en/liker-land-the-plaza-for-content-lovers/

File: src/layouts/index.vue [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            registration.unregister();
+          });
+        }
+      }
+      if (window.caches) {
+        const keyList = await window.caches.keys();
+        if (keyList?.length) {
+          await Promise.all(keyList.map(key => caches.delete(key)));
+        }
+      }
```
Comment (williamchong) [inline src/layouts/index.vue:146]:
```suggestion

      if (window.caches) {
        const keyList = await window.caches.keys();
        if (keyList?.length) {
          await Promise.all(
            keyList
              .filter(key => !key.startsWith('reader'))
              .map(key => caches.delete(key))
          );
        }
      }
```

File: src/locales/en.json [hunk 3]
```diff
@@ -336,6 +339,32 @@
...
+    {
+      "a": "每一本電子書都是獨一無二的，並由作者親自簽發。NFT書是可交易的電子書，您可以在Liker Land 書城的市場上將其出售或轉讓給其他人。這為您提供了資產轉換為現金的機會，或者與其他讀者進行交流和交易。",
+      "q": "NFT 電子書有什麼特別？"
+    },
+    {
+      "a": "成功購買電子書後，你會收到確認電郵。簡單幾步註冊帳户，作者便會把書發送到你的區塊鏈書櫃。你將擁有電子書的擁有權和內容訪問權限。",
+      "q": "購買成功後，會發生什麼事？"
+    },
+    {
+      "a": "網站跟一般電子書無異，主要支援 e-Pub 和 PDF 兩個格式。視乎作者設定，你可以彈性選擇在任何手機，平板，閱讀器線上瀏覽或者下載文本來讀。",
```
Comment (williamchong) [inline src/locales/en.json:357]:
```suggestion
      "a": "網站跟一般電子書無異，主要支援 EPUB 和 PDF 兩個格式。視乎作者設定，你可以彈性選擇在任何手機，平板，閱讀器線上瀏覽或者下載文本來讀。",

```

File: src/locales/zh-Hant.json [hunk 4]
```diff
@@ -336,6 +339,32 @@
...
   "identity_type_distributor": "經銷",
   "identity_type_publisher": "發行",
+  "index_bookstore_more_button": "查看更多",
+  "index_bookstore_title": "熱門書籍",
+  "index_category_button_browse_label": "瀏覽",
+  "index_category_button_ebook": "電子書",
+  "index_category_button_printed_book": "紙本書",
+  "index_faq_list": [
+    {
+      "a": "每一本電子書都是獨一無二的，並由作者親自簽發。NFT書是可交易的電子書，您可以在Liker Land 書城的市場上將其出售或轉讓給其他人。這為您提供了資產轉換為現金的機會，或者與其他讀者進行交流和交易。",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:349]:
```suggestion
      "a": "每一本電子書都是獨一無二的，並由作者親自簽發。NFT 書是可交易的電子書，您可以在 Liker Land 書城的市場上將其出售或轉讓給其他人。這為您提供了資產轉換為現金的機會，或者與其他讀者進行交流和交易。",

```

File: src/locales/zh-Hant.json [hunk 5]
```diff
@@ -336,6 +339,32 @@
...
+      "q": "NFT 電子書有什麼特別？"
+    },
+    {
+      "a": "成功購買電子書後，你會收到確認電郵。簡單幾步註冊帳户，作者便會把書發送到你的區塊鏈書櫃。你將擁有電子書的擁有權和內容訪問權限。",
+      "q": "購買成功後，會發生什麼事？"
+    },
+    {
+      "a": "網站跟一般電子書無異，主要支援 e-Pub 和 PDF 兩個格式。視乎作者設定，你可以彈性選擇在任何手機，平板，閱讀器線上瀏覽或者下載文本來讀。",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:357]:
```suggestion
      "a": "網站跟一般電子書無異，主要支援 EPUB 和 PDF 兩個格式。視乎作者設定，你可以彈性選擇在任何手機，平板，閱讀器線上瀏覽或者下載文本來讀。",

```

File: src/locales/zh-Hant.json [hunk 6]
```diff
@@ -336,6 +339,32 @@
...
+      "q": "買了 Liker Land 的電子書，可以如何讀？"
+    }
+  ],
+  "index_faq_title": "你會想知的事",
+  "index_features_title": "Liker Land的特色",
```
Comment (williamchong) [inline src/locales/zh-Hant.json:362]:
```suggestion
  "index_features_title": "Liker Land 的特色",

```

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/pages/index.vue [hunk 7]
```diff
@@ -1658,1043 +655,68 @@ export default {
...
     },
-    async fetchTrendingNFTList() {
-      try {
-        await this.lazyFetchLatestAndTrendingNFTClassIdList();
-      } catch (error) {
-        // eslint-disable-next-line no-console
-        console.error('Error occurred when fetching trending NFTs', error);
+    handleClickItem(event, item) {
+      if (item.isMultiple) {
+        event.stopPropagation();
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/pages/index.vue]: Updates, requires hacking for Vue router link
https://stackoverflow.com/questions/44815465/how-to-stop-router-link-from-sending-the-user-to-another-page

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/NFTBook/ItemCardV2.vue [hunk 8]
```diff
@@ -15,6 +15,7 @@
       :src="NFTImageUrl"
       :alt="NFTName"
       :to="nftCollectRoute"
+      :event="isLinkDisabled ? '' : 'click'"
```
[the remark this answers is not in the document]
Comment (nwingt) [reply src/components/NFTBook/ItemCardV2.vue:18]: Related to this, cannot use preventDefault to stop navigation without setting event to empty string
https://github.com/likecoin/liker-land/pull/1586#discussion_r1582632883

**Review by williamchong — APPROVED** [review]
Please resolve conflict

**Review by AuroraHuang22 — APPROVED** [review]

File: src/pages/index.vue [hunk 9]
```diff
@@ -880,18 +892,81 @@ export default {
       if (!this.$refs.hero) return;
 
       const { clientHeight: heroHeight } = this.$refs.hero;
-      if (window.scrollY > heroHeight) {
```
Comment (williamchong) [inline src/pages/index.vue]: Just set `isHeaderFixed = window.scrollY > heroHeight`?

File: src/static/images/og/default-zh.JPG
Comment (williamchong) [inline src/static/images/og/default-zh.JPG:1]: Extension is JPG instead of jpg?

File: src/pages/index.vue (hunk 9 above)
Comment (nwingt) [reply src/pages/index.vue]: Updated

File: src/static/images/og/default-zh.JPG
Comment (nwingt) [reply src/static/images/og/default-zh.JPG:1]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**